### ISSUE TITLE
Preserve notifications untill manually marked as resolved

### ISSUE THREAD

**Issue by lundibundi** [sindresorhus/refined-github#1481]
I'd love to see this behavior (similar to what GitLab has with its Todo section) when notifications you `Mark as unread` will not be cleared from notifications list until you manually 'resolve' them. Or maybe add some sort of Pinned issues/PRs that you will be able to use to keep track of issues/PRs etc you want.

Thanks for the great extension. Hopefully, I didn't miss a similar issue in already opened/closed ones.

**Comment by bfred-it** [comment]
Sounds like you want a “Watch” feature. It should be it’s own extension, if it doesn’t already exist

**Comment by lundibundi** [comment]
Well, I haven't found one unfortunately and using bookmarks is kind of complex and hard to follow through thanks for a suggestion. 
I thought it will maybe be possible to add configuration 'don't check notification as read upon visit' and a button `Mark as read`?

**Comment by bfred-it** [comment]
Thanks for the question, but I personally wouldn't.

**Comment by sindresorhus** [comment]
We have decided to pass on this. While it could be useful for some we feel it's not generally useful enough to warrant the implementation complexity. The current "mark as unread" functionality is already quite complex and we would prefer to not add even more bloat to it. GitHub is an ever moving platform and every feature we add means a lot of future maintenance work when GitHub changes stuff.
